src/SW_Weather.c

```diff
@@ -502,29 +503,81 @@ void driestQtrSouthAdjMonYears(int month, int *adjustedYearZero, int *adjustedYe
  @param[in] use_weathergenerator_only A boolean; if `swFALSE`, code attempts to
    read weather files from disk.
  @param[in] weather_prefix File name of weather data without extension.
+ @param[in] use_cloudCoverMonthly A boolean; if `swTRUE`, function will interpolate

```

Comment:
Please make documentation more precise for all three `use_*Monthly`
- `"read-in from disk"` is not correct for the scope of this function
- instead, `"... interpolate mean monthly values provided by \ref cloudcov to daily time series"`